scripts/manage_subscription.py

```diff
@@ -104,76 +86,66 @@ def pay_slots(account, network, domain, subscription_contract, encryptor_slots):
     click.echo(f"Connected to {network.name} network.")
     transactor = Transactor(account=account)
     subscription_contract = registry.get_contract(
-        contract_name=subscription_contract,
-        domain=domain
+        contract_name=subscription_contract, domain=domain
     )
     erc20 = Contract(subscription_contract.feeToken())
-    fee = _calculate_slot_fees(
-        subscription_contract=subscription_contract,
-        slots=encryptor_slots
-    )
-    _erc20_approve(
-        amount=fee,
-        erc20=erc20,
-        receiver=subscription_contract,
-        transactor=transactor
-    )
+    fee = _calculate_slot_fees(subscription_contract=subscription_contract, slots=encryptor_slots)
+    _erc20_approve(amount=fee, erc20=erc20, receiver=subscription_contract, transactor=transactor)
     click.echo(f"Paying for {encryptor_slots} new encryptor slots.")
-    transactor.transact(
-        subscription_contract.payForEncryptorSlots,
-        encryptor_slots
-    )
+    transactor.transact(subscription_contract.payForEncryptorSlots, encryptor_slots)
 
 
 @cli.command(cls=ConnectedProviderCommand)
 @account_option()
 @network_option(required=True)
 @domain_option
 @ritual_id_option
-@access_controller_option
 @encryptors_option
-def add_encryptors(account, network, domain, ritual_id, access_controller, encryptors):
+def add_encryptors(account, network, domain, ritual_id, encryptors):
     """Authorize encryptors to the access control contract for a ritual."""
     click.echo(f"Connected to {network.name} network.")
-    access_controller = registry.get_contract(
-        contract_name=access_controller,
-        domain=domain
-    )
+
+    # lookup the access controller + authority for the ritual
+    coordinator = registry.get_contract(contract_name="Coordinator", domain=domain)
+    ritual = coordinator.rituals(ritual_id)
+    access_controller = Contract(ritual.access_controller)  # uses polygonscan API
+    if account != ritual.authority:
```

Comment:
Is this supposed to be (?):
```suggestion
    if account.address != ritual.authority:
```